packages/shared/src/components/cards/v1/SharePostCard.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { forwardRef, ReactElement, Ref, useRef, useState } from 'react';
 import ActionButtons from './ActionButtons';
 import { SharedPostText } from '../SharedPostText';
-import { SharedPostCardFooter } from '../SharedPostCardFooter';
+import { SharedPostCardFooter } from './SharedPostCardFooter';
```

Comment:
Not on this line, but further down, can't comment there, we need to make sure we follow the rule for summary to not be more then 300 chars here as well.